### PR TITLE
2.8.1 release

### PR DESCRIPTION
**Pull request by hpk42**

### CODE REVIEW

**Comment by hpk42** [comment]
release files are at https://devpi.net/hpk/dev/pytest/2.8.1

File: _pytest/pytester.py [hunk 1]
```diff
@@ -534,6 +534,20 @@ def syspathinsert(self, path=None):
...
             path = self.tmpdir
         sys.path.insert(0, str(path))
+        # a call to syspathinsert() usually means that the caller
+        # wants to import some dynamically created files.
+        # with python3 we thus invalidate import caches.
+        self._possibly_invalidate_import_caches()
+
+    def _possibly_invalidate_import_caches(self):
+        # invalidate caches if we can (py33 and above)
+        try:
```
Comment (RonnyPfannschmidt) [inline _pytest/pytester.py]: cant we use `from importlib import invalidate_caches` to avoid the feature check in the else branch?

**Comment by RonnyPfannschmidt** [comment]
:+1:

File: _pytest/pytester.py (hunk 1 above)
Comment (hpk42) [inline _pytest/pytester.py]: probably -- but it's minor -- wouldn't want to repack/test everything for that.

Comment (The-Compiler) [inline _pytest/pytester.py]: Or just do `importlib.invalidate_caches()` here and do `except (ImportError, AttributeError): pass`

Comment (RonnyPfannschmidt) [inline _pytest/pytester.py]: @The-Compiler nope ^^

Comment (hpk42) [inline _pytest/pytester.py]: ronny regards it as a matter of principle: if ``invalidate_caches()`` raises an AttributeError you would swallow it :)

File: doc/en/cache.rst [hunk 2]
```diff
@@ -73,7 +73,7 @@ If you then run it with ``--lf``::
 
     $ py.test --lf
     ======= test session starts ========
-    platform linux -- Python 3.4.2, pytest-2.8.1.dev1, py-1.4.30, pluggy-0.3.1
+    platform linux -- Python 3.4.3, pytest-2.8.1, py-1.4.30, pluggy-0.3.1
```
Comment (nicoddemus) [inline doc/en/cache.rst]: No need to update this, just thought I would mention: next time we generate the examples I think (and Ronny agreess) that we should use Python 3.5 as a rule of thumb.

**Comment by nicoddemus** [comment]
LGTM! :+1: 

Shame #1058 didn't make it though.

**Comment by hpk42** [comment]
we can update the docs on pytest.org after #1058 made it :)

**Comment by nicoddemus** [comment]
OK! :smile:

**Pull request merged**